package.json

```diff
@@ -339,6 +339,23 @@
         "category": "OpenShift"
       }
     ],
+    "keybindings": [
+      {
+        "command": "openshift.explorer.login",
+        "key": "ctrl+shift+l",
+        "mac": "cmd+shift+l"
+      },
+      {
+        "command": "openshift.explorer.refresh",
```

Comment:
@sudhirverma  could you check if it is possible to activate this only for OpenShift Views Container? Otherwise the result of this command called with key combination would not be visible to the user.